python/DYNAMODB_TABLE_ENCRYPTED_KMS/DYNAMODB_TABLE_ENCRYPTED_KMS_test.py

```diff
@@ -0,0 +1,143 @@
+import unittest
+from mock import patch, MagicMock
+from rdklib import Evaluation, ComplianceType, InvalidParametersError
+import rdklibtest
+
+# Define the default resource to report to Config Rules
+RESOURCE_TYPE = 'AWS::DynamoDB::Table'
+MODULE = __import__('DYNAMODB_TABLE_ENCRYPTED_KMS')
+RULE = MODULE.DYNAMODB_TABLE_ENCRYPTED_KMS()
+CLIENT_FACTORY = MagicMock()
+DB_CLIENT_MOCK = MagicMock()
+CONFIG_CLIENT = MagicMock()
+
+def mock_get_client(service, *args, **kwargs):
+    if service == 'dynamodb':
+        return DB_CLIENT_MOCK
+    if service == 'config':
+        return CONFIG_CLIENT
+    raise Exception("Attempting to create an unknown client")
+
+@patch.object(CLIENT_FACTORY, 'build_client', MagicMock(side_effect=mock_get_client))
+class ComplianceTest(unittest.TestCase):
+    MOCK_CONF_ITEM = {"configuration": {"tableName": "testNonEncrypt", "tableStatus": "ACTIVE", "ssedescription": {"status": "ENABLED", "ssetype": "KMS", "kmsmasterKeyArn": "arn:aws:kms:ap-southeast-2:437313072050:key/f4fb52b5-03a0-4397-a2db-cb5b94abb0a6"}}}
+    MOCK_CONF_ITEM_NON = {"configuration": {"tableName": "table123", "tableStatus": "ACTIVE"}}
+    MOCK_CONF_ITEM_NA = {"configuration": {"tableName": "testNA", "tableStatus": "DELETED"}}
+    MOCK_NON_COMP = {"Table": {"TableArn": "arn:aws:dynamodb:ap-southeast-2:437313072050:table/testNonEncrypt", "ItemCount": 0, "CreationDateTime": 1583296041.861}}
+    MOCK_COMP = {"Table": {"TableArn": "arn:aws:dynamodb:ap-southeast-2:437313072050:table/test123", "SSEDescription": {"Status": "UPDATING", "KMSMasterKeyArn": "arn:aws:kms:ap-southeast-2:437313072050:key/f4fb52b5-03a0-4397-a2db-cb5b94abb0a6", "SSEType": "KMS"}}}
+
+    def test_emptyruleparameter_returnsuccess(self):
+        rule_invalid_parameter = {
+            "KmsKeyArns":  ""
+        }
+        response = RULE.evaluate_parameters(rule_invalid_parameter)
+        self.assertEqual(response, {})
+
+    def test_invalidruleparameter_returnserror(self):
+        rule_invalid_parameter = {
+            "KmsKeyArns":  "invalid-arn,arn:aws:kms:ap-southeast-2:437313072050:key/f4fb52b5-03a0-4397-a2db-cb5b94abb0a6"
+        }
+        with self.assertRaises(InvalidParametersError) as context:
+            RULE.evaluate_parameters(rule_invalid_parameter)
+        self.assertIn('Invalid AWS KMS Key Arn format for "invalid-arn". AWS KMS Key Arn starts with "arn:aws:kms:"', str(context.exception))
+
+    def test_scenario1_evaluateparameters_validruleparameter_returnsuccess(self):
```

Comment:
Follow the syntax for test function names test_<scenarioName>_<methodName>_<inputconditions>_<outputResult>

Reply:
added test cases name starts with test_